src/main/java/net/minestom/server/UpdateManager.java

```diff
@@ -39,8 +39,8 @@
 
     {
         // DEFAULT THREAD PROVIDER
-        //threadProvider = new PerGroupChunkProvider();
         threadProvider = new PerInstanceThreadProvider();
+        //threadProvider = new PerChunkThreadProvider();
```

Comment:
This is a breaking change. I rely on the comment being above not below - this will break my workflow please revert.

Reply:
Sorry, this must be done for the future of the project